src/data-collector/kount.js

```diff
@@ -92,11 +92,12 @@ Kount.prototype._setupIFrame = function () {
   iframe.style.position = 'fixed';
   iframe.style.left = '-999999px';
   iframe.style.top = '-999999px';
+  iframe.title = IFRAME_ID_PREFIX + this._deviceSessionId;
 
   document.body.appendChild(iframe);
   setTimeout(function () {
     iframe.src = self._currentEnvironment.url + '/logo.htm' + params;
-    iframe.innerHTML = '<img src="' + self._currentEnvironment.url + '/logo.gif' + params + '" />';
+    iframe.innerHTML = '<img src="' + self._currentEnvironment.url + '/logo.gif' + params + '" alt="" />';
```

Comment:
I'm surprised that an empty `alt` attribute satisfies your accessibility linter.

Reply:
Every img tag in the page should have an alt tag so it's screaming about this one!